src/Microsoft.Health.Dicom.Api/Extensions/WorkitemResponseStatusExtensions.cs

```diff
@@ -37,6 +37,15 @@ public static class WorkitemResponseStatusExtensions
             { WorkitemResponseStatus.PartialContent, HttpStatusCode.PartialContent },
         };
 
+    private static readonly IReadOnlyDictionary<WorkitemResponseStatus, HttpStatusCode> ChangeStateResponseStatusToHttpStatusCodeMapping =
+        new Dictionary<WorkitemResponseStatus, HttpStatusCode>()
+        {
+            { WorkitemResponseStatus.Success, HttpStatusCode.OK },
+            { WorkitemResponseStatus.Failure, HttpStatusCode.BadRequest },
+            { WorkitemResponseStatus.NoContent, HttpStatusCode.NotFound },
```

Comment:
I didnt see the NoContent being mentioned in the spec(https://dicom.nema.org/medical/dicom/current/output/html/part18.html#table_11.7.3-1). Do you see somewhere?